### DISCUSSION TITLE
How to include weblink in an animation shortcode?

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [mcanouil/quarto-animate#2] [Q&A] [answered]
Hi there @mcanouil 

Thanks for porting this extension into Quarto. I'm trying to animate a text which has a link in-words, for instance:

```
{{< animate fadeInLeft  "Hey there, this is simple text. This is a link [book](https://camilogarciabotero.github.io/biocomp-book/)." >}}
```

The animation works fine, but the output will lack the link. How should I pass a text with links without losing the link-generation.?

Thanks in advance.

**Comment by mcanouil** [comment, marked as answer]
_The following was added to README and example document._

To note, animating clickable elements is not really something anyone should do.

The following won't work:

```markdown
{{< animate bounce "[HTML](https://m.canouil.fr/quarto-animate/)" >}}
```

But this will:

```markdown
[[HTML](https://m.canouil.fr/quarto-animate/)]{.animate__animated .animate__bounce style="display:inline-block;"}
```

Or:

```markdown
::: {.animate__animated .animate__bounce}
[HTML](https://m.canouil.fr/quarto-animate/)
:::
```

See <https://animate.style/> for more details.

PS: no need to ping when creating issues/discussions/PRs. 😉

**Comment by camilogarciabotero** [comment]
Thanks for the answer. The option
```
::: {.animate__animated .animate__bounce}
[HTML](https://m.canouil.fr/quarto-animate/)
:::
```
Did work for me. Thanks also for the suggestions!
